scalate-jaxrs/src/main/scala/org/fusesource/scalate/rest/ScalateTemplateProvider.scala

```diff
@@ -103,8 +103,6 @@ class ScalateTemplateProvider extends MessageBodyWriter[AnyRef] {
         var notFound = true
         for (uri <- ServletHelper.errorUris() if notFound) {
           try {
-            val template = engine.load(uri)
```

Comment:
Is there any side effect `TemplateEngine#load` ?

https://github.com/scalate/scalate/blob/cf08ee72c7313429d96a9883efb5cebf980be753/scalate-core/src/main/scala/org/fusesource/scalate/TemplateEngine.scala#L411-L462

Reply:
Oh, as you pointed out.
I should just delete the variable that will receive the return value.

Reply:
https://github.com/scalate/scalate/pull/123